dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -440,6 +449,7 @@ private static void checkForExistingDOIForWorkflowVersion(WorkflowVersion workfl
      * Get an existing Zenodo DOI for the workflow if one exists
      * otherwise return null
      * @param workflow workflow
+     * @return the DOI for a workflow or null
```

Comment:
Since it's returning an Optional, it never returns null

Reply:
fixed comment